vsphere/datadog_checks/vsphere/legacy/vsphere_legacy.py

```diff
@@ -0,0 +1,993 @@
+# (C) Datadog, Inc. 2018-2019
```

Comment:
I think it's important to keep the commit history, especially for a file big like this one.

One way to achieve that is to keep the original file name `legacy/vsphere.py` and name the new implementation  `vsphere_new.py`.

We can if needed, in another PR rename like this to get the expected result:
- `legacy/vsphere.py => `legacy/vsphere_legacy.py`
- `vsphere_new.py => `vsphere.py`

I think this little effort is worthwhile to keep the commit history that can be helpful for people reading this code later on.